src/ontology/cl-edit.owl

```diff
@@ -32426,6 +32428,17 @@ AnnotationAssertion(terms:date obo:CL_4047012 "2024-09-06T08:41:39Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047012 "angiogenic pericyte"@en)
 SubClassOf(obo:CL_4047012 obo:CL_0000669)
 
+# Class: obo:CL_4047020 (epithelial stem cell of gut)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24326621") Annotation(oboInOwl:hasDbXref "PMID:38835047") obo:IAO_0000115 obo:CL_4047020 "An epithelial stem cell located in a stem cell niche at the base of intestinal crypts. This cell is capable of self-renewal and multipotent differentiation into all intestinal epithelial cell types including enterocytes, goblet cells, enteroendocrine cells, and paneth cells. These cells divide rapidly to maintain epithelial homeostatis. A variety of adjacent cell types secrete signals that support and maintain the identity of this cell type in the niche, including Paneth Cells and specialised fibroblasts.
+Expresses marker LGR5 in humans and markers ASCL2 and SMOC2 in mice.")
```

Comment:
It seems that LGR5 is not only expressed in humans but also in mice PMID: [32705632](https://pubmed.ncbi.nlm.nih.gov/32705632), [PMID: 24326621](https://pubmed.ncbi.nlm.nih.gov/24326621/)